
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Tasks(object):
    def setupUi(self, Tasks):
        Tasks.setObjectName("Tasks")
        Tasks.resize(1028, 626)
        Tasks.setMaximumSize(QtCore.QSize(1028, 626))
        palette = QtGui.QPalette()
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 85, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 170, 127))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Light, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 127, 63))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Midlight, brush)
        brush = QtGui.QBrush(QtGui.QColor(127, 42, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Dark, brush)
        brush = QtGui.QBrush(QtGui.QColor(170, 56, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Mid, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Text, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.BrightText, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.ButtonText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Base, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 85, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.Shadow, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 170, 127))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.AlternateBase, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 220))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.ToolTipBase, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Active, QtGui.QPalette.ToolTipText, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 85, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 170, 127))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Light, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 127, 63))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Midlight, brush)
        brush = QtGui.QBrush(QtGui.QColor(127, 42, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Dark, brush)
        brush = QtGui.QBrush(QtGui.QColor(170, 56, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Mid, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Text, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.BrightText, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.ButtonText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Base, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 85, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.Shadow, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 170, 127))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.AlternateBase, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 220))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.ToolTipBase, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Inactive, QtGui.QPalette.ToolTipText, brush)
        brush = QtGui.QBrush(QtGui.QColor(127, 42, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.WindowText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 85, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Button, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 170, 127))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Light, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 127, 63))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Midlight, brush)
        brush = QtGui.QBrush(QtGui.QColor(127, 42, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Dark, brush)
        brush = QtGui.QBrush(QtGui.QColor(170, 56, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Mid, brush)
        brush = QtGui.QBrush(QtGui.QColor(127, 42, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Text, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 255))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.BrightText, brush)
        brush = QtGui.QBrush(QtGui.QColor(127, 42, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.ButtonText, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 85, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Base, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 85, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Window, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.Shadow, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 85, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.AlternateBase, brush)
        brush = QtGui.QBrush(QtGui.QColor(255, 255, 220))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.ToolTipBase, brush)
        brush = QtGui.QBrush(QtGui.QColor(0, 0, 0))
        brush.setStyle(QtCore.Qt.SolidPattern)
        palette.setBrush(QtGui.QPalette.Disabled, QtGui.QPalette.ToolTipText, brush)
        Tasks.setPalette(palette)
        self.line_5 = QtWidgets.QFrame(Tasks)
        self.line_5.setGeometry(QtCore.QRect(640, 0, 20, 621))
        self.line_5.setFrameShape(QtWidgets.QFrame.VLine)
        self.line_5.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line_5.setObjectName("line_5")
        self.line_6 = QtWidgets.QFrame(Tasks)
        self.line_6.setGeometry(QtCore.QRect(0, 50, 1001, 20))
        self.line_6.setFrameShape(QtWidgets.QFrame.HLine)
        self.line_6.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line_6.setObjectName("line_6")
        self.config_services_label_2 = QtWidgets.QLabel(Tasks)
        self.config_services_label_2.setGeometry(QtCore.QRect(740, 0, 161, 51))
        font = QtGui.QFont()
        font.setPointSize(13)
        font.setBold(True)
        font.setWeight(75)
        self.config_services_label_2.setFont(font)
        self.config_services_label_2.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.config_services_label_2.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.config_services_label_2.setAlignment(QtCore.Qt.AlignCenter)
        self.config_services_label_2.setObjectName("config_services_label_2")
        self.osversion_label = QtWidgets.QLabel(Tasks)
        self.osversion_label.setGeometry(QtCore.QRect(130, 140, 111, 20))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.osversion_label.setFont(font)
        self.osversion_label.setAlignment(QtCore.Qt.AlignCenter)
        self.osversion_label.setObjectName("osversion_label")
        self.osversion_text = QtWidgets.QTextEdit(Tasks)
        self.osversion_text.setGeometry(QtCore.QRect(130, 160, 111, 41))
        self.osversion_text.setObjectName("osversion_text")
        self.routetype_label = QtWidgets.QLabel(Tasks)
        self.routetype_label.setGeometry(QtCore.QRect(270, 70, 91, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.routetype_label.setFont(font)
        self.routetype_label.setAlignment(QtCore.Qt.AlignCenter)
        self.routetype_label.setObjectName("routetype_label")
        self.routetype_text = QtWidgets.QTextEdit(Tasks)
        self.routetype_text.setGeometry(QtCore.QRect(260, 90, 111, 41))
        self.routetype_text.setObjectName("routetype_text")
        self.address_devices_label = QtWidgets.QLabel(Tasks)
        self.address_devices_label.setGeometry(QtCore.QRect(250, 140, 131, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.address_devices_label.setFont(font)
        self.address_devices_label.setAlignment(QtCore.Qt.AlignCenter)
        self.address_devices_label.setObjectName("address_devices_label")
        self.addressdevices_text = QtWidgets.QTextEdit(Tasks)
        self.addressdevices_text.setGeometry(QtCore.QRect(260, 160, 111, 41))
        self.addressdevices_text.setObjectName("addressdevices_text")
        self.interfacelist_label = QtWidgets.QLabel(Tasks)
        self.interfacelist_label.setGeometry(QtCore.QRect(520, 140, 111, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.interfacelist_label.setFont(font)
        self.interfacelist_label.setAlignment(QtCore.Qt.AlignCenter)
        self.interfacelist_label.setObjectName("interfacelist_label")
        self.interfacelist_text = QtWidgets.QTextEdit(Tasks)
        self.interfacelist_text.setGeometry(QtCore.QRect(520, 160, 111, 41))
        self.interfacelist_text.setObjectName("interfacelist_text")
        self.commands_label = QtWidgets.QLabel(Tasks)
        self.commands_label.setGeometry(QtCore.QRect(20, 230, 91, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.commands_label.setFont(font)
        self.commands_label.setAlignment(QtCore.Qt.AlignCenter)
        self.commands_label.setObjectName("commands_label")
        self.commands_text = QtWidgets.QTextEdit(Tasks)
        self.commands_text.setGeometry(QtCore.QRect(130, 220, 501, 41))
        self.commands_text.setObjectName("commands_text")
        self.typepolicy_label = QtWidgets.QLabel(Tasks)
        self.typepolicy_label.setGeometry(QtCore.QRect(400, 70, 91, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.typepolicy_label.setFont(font)
        self.typepolicy_label.setAlignment(QtCore.Qt.AlignCenter)
        self.typepolicy_label.setObjectName("typepolicy_label")
        self.typepolicy_text = QtWidgets.QTextEdit(Tasks)
        self.typepolicy_text.setGeometry(QtCore.QRect(390, 90, 111, 41))
        self.typepolicy_text.setObjectName("typepolicy_text")
        self.typeservice_label = QtWidgets.QLabel(Tasks)
        self.typeservice_label.setGeometry(QtCore.QRect(530, 70, 91, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.typeservice_label.setFont(font)
        self.typeservice_label.setAlignment(QtCore.Qt.AlignCenter)
        self.typeservice_label.setObjectName("typeservice_label")
        self.typeservice_text = QtWidgets.QTextEdit(Tasks)
        self.typeservice_text.setGeometry(QtCore.QRect(520, 90, 111, 41))
        self.typeservice_text.setObjectName("typeservice_text")
        self.configs_label = QtWidgets.QLabel(Tasks)
        self.configs_label.setGeometry(QtCore.QRect(20, 290, 71, 21))
        font = QtGui.QFont()
        font.setPointSize(11)
        self.configs_label.setFont(font)
        self.configs_label.setAlignment(QtCore.Qt.AlignCenter)
        self.configs_label.setObjectName("configs_label")
        self.serviceconfig_text = QtWidgets.QTextEdit(Tasks)
        self.serviceconfig_text.setGeometry(QtCore.QRect(130, 280, 501, 41))
        self.serviceconfig_text.setObjectName("serviceconfig_text")
        self.output_output_label = QtWidgets.QLabel(Tasks)
        self.output_output_label.setGeometry(QtCore.QRect(750, 70, 151, 21))
        font = QtGui.QFont()
        font.setPointSize(14)
        self.output_output_label.setFont(font)
        self.output_output_label.setAlignment(QtCore.Qt.AlignCenter)
        self.output_output_label.setObjectName("output_output_label")
        self.output_output_text = QtWidgets.QTextEdit(Tasks)
        self.output_output_text.setGeometry(QtCore.QRect(670, 90, 321, 481))
        self.output_output_text.setObjectName("output_output_text")
        self.comboBox = QtWidgets.QComboBox(Tasks)
        self.comboBox.setGeometry(QtCore.QRect(170, 20, 111, 31))
        self.comboBox.setObjectName("comboBox")
        self.comboBox.addItem("")
        self.comboBox.setItemText(0, "")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.switching_label_2 = QtWidgets.QLabel(Tasks)
        self.switching_label_2.setGeometry(QtCore.QRect(10, 10, 131, 51))
        font = QtGui.QFont()
        font.setPointSize(13)
        font.setBold(True)
        font.setWeight(75)
        self.switching_label_2.setFont(font)
        self.switching_label_2.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.switching_label_2.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.switching_label_2.setAlignment(QtCore.Qt.AlignCenter)
        self.switching_label_2.setObjectName("switching_label_2")
        self.brand_label = QtWidgets.QLabel(Tasks)
        self.brand_label.setGeometry(QtCore.QRect(20, 70, 71, 20))
        font = QtGui.QFont()
        font.setPointSize(10)
        font.setBold(False)
        font.setWeight(50)
        self.brand_label.setFont(font)
        self.brand_label.setAlignment(QtCore.Qt.AlignCenter)
        self.brand_label.setObjectName("brand_label")
        self.brand_text = QtWidgets.QTextEdit(Tasks)
        self.brand_text.setGeometry(QtCore.QRect(10, 90, 101, 41))
        self.brand_text.setObjectName("brand_text")
        self.devicetype_label = QtWidgets.QLabel(Tasks)
        self.devicetype_label.setGeometry(QtCore.QRect(10, 140, 91, 20))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.devicetype_label.setFont(font)
        self.devicetype_label.setAlignment(QtCore.Qt.AlignCenter)
        self.devicetype_label.setObjectName("devicetype_label")
        self.devicetype_text = QtWidgets.QTextEdit(Tasks)
        self.devicetype_text.setGeometry(QtCore.QRect(10, 160, 101, 41))
        self.devicetype_text.setObjectName("devicetype_text")
        self.serveraddress_label = QtWidgets.QLabel(Tasks)
        self.serveraddress_label.setGeometry(QtCore.QRect(130, 70, 111, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.serveraddress_label.setFont(font)
        self.serveraddress_label.setAlignment(QtCore.Qt.AlignCenter)
        self.serveraddress_label.setObjectName("serveraddress_label")
        self.serveraddress_text = QtWidgets.QTextEdit(Tasks)
        self.serveraddress_text.setGeometry(QtCore.QRect(130, 90, 111, 41))
        self.serveraddress_text.setObjectName("serveraddress_text")
        self.pathfile_label = QtWidgets.QLabel(Tasks)
        self.pathfile_label.setGeometry(QtCore.QRect(400, 140, 91, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.pathfile_label.setFont(font)
        self.pathfile_label.setAlignment(QtCore.Qt.AlignCenter)
        self.pathfile_label.setObjectName("pathfile_label")
        self.pathfile_text = QtWidgets.QTextEdit(Tasks)
        self.pathfile_text.setGeometry(QtCore.QRect(390, 160, 111, 41))
        self.pathfile_text.setObjectName("pathfile_text")

        self.retranslateUi(Tasks)
        QtCore.QMetaObject.connectSlotsByName(Tasks)

    def retranslateUi(self, Tasks):
        _translate = QtCore.QCoreApplication.translate
        Tasks.setWindowTitle(_translate("Tasks", "Tasks Page"))
        self.config_services_label_2.setText(_translate("Tasks", "Output of Task"))
        self.osversion_label.setText(_translate("Tasks", "OS Version"))
        self.routetype_label.setText(_translate("Tasks", "Route Type"))
        self.address_devices_label.setText(_translate("Tasks", "Address Devices"))
        self.interfacelist_label.setText(_translate("Tasks", "Interface List"))
        self.commands_label.setText(_translate("Tasks", "Commands"))
        self.typepolicy_label.setText(_translate("Tasks", "Type Policy"))
        self.typeservice_label.setText(_translate("Tasks", "Type Service"))
        self.configs_label.setText(_translate("Tasks", "Configs"))
        self.output_output_label.setText(_translate("Tasks", "OutPut"))
        self.comboBox.setItemText(1, _translate("Tasks", "Update/OS"))
        self.comboBox.setItemText(2, _translate("Tasks", "Update/IPS"))
        self.comboBox.setItemText(3, _translate("Tasks", "Update/AV"))
        self.comboBox.setItemText(4, _translate("Tasks", "Layer 2 Config"))
        self.comboBox.setItemText(5, _translate("Tasks", "Routing Config"))
        self.comboBox.setItemText(6, _translate("Tasks", "Security Config"))
        self.comboBox.setItemText(7, _translate("Tasks", "Config Service"))
        self.switching_label_2.setText(_translate("Tasks", "Task Type"))
        self.brand_label.setText(_translate("Tasks", "Brand"))
        self.devicetype_label.setText(_translate("Tasks", "Device Type"))
        self.serveraddress_label.setText(_translate("Tasks", "Server Address"))
        self.pathfile_label.setText(_translate("Tasks", "Path File"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Tasks = QtWidgets.QWidget()
    ui = Ui_Tasks()
    ui.setupUi(Tasks)
    Tasks.show()
    sys.exit(app.exec_())
